omnibus/config/software/sds.rb

```diff
@@ -0,0 +1,20 @@
+name "sds"
+
+default_version "042de62f5a24fbceb4f4849256c3ee5c005b7057"
+source git: 'https://github.com/DataDog/dd-sensitive-data-scanner'
+
+build do
+    license "Apache-2.0"
+    license_file "./LICENSE"
+
+    # no Windows support for now.
+    if linux_target? || osx_target?
+        command "cargo build --release", cwd: "#{project_dir}/sds-go/rust"
+        if osx_target?
+            command "cp sds-go/rust/target/release/libsds_go.dylib #{install_dir}/embedded/lib"
```

Comment:
This will cause omnibus to warn and insist that you use the dedicated `copy` DSL method

Reply:
Should be addressed.